### DISCUSSION TITLE
Why "BiasAdjustCXX"?

### DISCUSSION THREAD

**Discussion by muhshif** [btschwertfeger/python-cmethods#29] [Q&A]
Why is the other library "BiasAdjustCXX" introduced separately instead of injecting it to the updated version of "CMethods"?

**Comment by btschwertfeger** [comment]
The BiasAdjustCXX tool (https://github.com/btschwertfeger/BiasAdjustCXX) is not related to CMethods. The "C" in "CMethods" stands for "correction". While python-cmethods is a pure Python library, BiasAdjustCXX is written in C++ and can be executed from the command line. I am currently enhancing the installation and testing process for BiasAdjustCXX and plan to release a new version later this week.

For adjusting large datasets, I recommend using BiasAdjustCXX as it is significantly faster than the Python module (see this performance comparison: https://github.com/btschwertfeger/BiasAdjustCXX-Performance-Test/blob/master/index.ipynb). However, the Python module is still a valuable resource and can serve as a starting point for understanding and applying various bias correction techniques an smaller data sets.
